package.json

```diff
@@ -44,7 +44,7 @@
     "lodash": "^3.10.0",
     "method-override": "^2.3.3",
     "mocha": "~1.20.0",
-    "mongoose": "^4.0.6",
+    "mongoose": "4.1.11",
```

Comment:
@codydaig which change exactly caused this issue. I don't see anything in their changelog that is marked as breaking or very relevant to the error. Do you know of a specific issue in the Mongoose repo?